### PR TITLE
Added fee delegation feature

### PR DESCRIPTION
**Pull request by praveen-klaytn**
As part of fee delegation boiler plate, deployer signs a sample contract from frontend application and backend will perform feepayer transaction , submission of transaction

Added opensource guidelines to the repo

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/--bug-report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Attachments**
+If applicable, add data, logs or screenshots to help explain your problem.
+
+**Environment (please complete the following information)**
+ - RPC url's used or node version that exhibits the issue
+ - The version pragma of the smart contract(s) involved
```
Comment (toniya-klaytn) [inline .github/ISSUE_TEMPLATE/--bug-report.md:24]:
```suggestion
 - Which module is being used. eg: sign
```

**Comment by toniya-klaytn** [comment]
@praveen-klaytn  Also please refactor the existing .gitignore to the below 
```
# depedancies
*/node_modules
*/*/node_modules/.bin/
*/*/.next
```

**Comment by praveen-klaytn** [comment]
> @praveen-klaytn Also please refactor the existing .gitignore to the below
> 
> ```
> # depedancies
> */node_modules
> */*/node_modules/.bin/
> */*/.next
> ```

Update gitignore related to feedelegation in root path gitignore file

File: feedelegation/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# FeeDelegation using web3modal for Baobab network
+
+Steps to run the application
+`cd feedelegation`
+
+1. **Setup Application**
+    - `yarn install`
+    - Modify .env details in feedelegation-be
```
[the remark this answers is not in the document]
Comment (praveen-klaytn) [reply feedelegation/README.md]: moved the .env to fee-delegation root folder

File: feedelegation/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Steps to run the application
+`cd feedelegation`
+
+1. **Setup Application**
+    - `yarn install`
+    - Rename .env.example and update privatekey/publickey details of any account from metamask or kaikas wallet for fee payer containing test klay coins
+    - `yarn start`
+    - Open application in browser `http://localhost:3000`
+2. Fee Delegates a contract deployment with a deployer from wallet connected to Baobab from UI and fee payer from Backend
```
Comment (toniya-klaytn) [inline feedelegation/README.md:11]:
```suggestion
3. Connect to Baobab testnet 
4. Sign the transaction from deployer address and delegate the fee to admin(fee payer) 
5. On Deploy, Admin who is the fee payer will deploy the contract by paying for the transaction.
```

File: feedelegation/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+3. Fee Delegation
```
Comment (toniya-klaytn) [inline feedelegation/README.md:12]:
```suggestion
```

File: feedelegation/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+	a) Deployer Signature from UI
```
Comment (toniya-klaytn) [inline feedelegation/README.md:13]:
```suggestion
```

File: feedelegation/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+	b) Fee Payer Signature and Transaction submission from Backend
```
Comment (toniya-klaytn) [inline feedelegation/README.md:14]:
```suggestion
```

File: feedelegation/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# FeeDelegation using web3modal for Baobab network
```
Comment (toniya-klaytn) [inline feedelegation/README.md:1]: Are we using web3modal here ? I see its directly connecting to kaikas without using web3modal

File: feedelegation/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (toniya-klaytn) [inline feedelegation/README.md:2]:
```suggestion
In order to Improve the usability of our blockchain, Klaytn has implemented on-chain fee delegation. It means that the service providers will be paying for the transaction fees on behalf of the users.

Klaytn-Dapp-Kit has the boiler plate code to implement fee delegation. More details on the Fee Delegation mechanism can be found here : https://medium.com/klaytn/klaytn-usability-improvement-series-3-fee-delegation-69b286c11968
![0_FsymS6h3grVDxHBv](https://user-images.githubusercontent.com/100742846/207226672-5cad2bcd-8d20-4480-a0c4-e2054ff41959.png)

This code has an example of deploying a contract without paying for the Txn as a user. It has 
	a) A UI where the user can sign the contract deployment and delegate the fee to admin
	b) A backend where the admin can pay for the transaction and execute. 

**UI**
An example smart contract, abi and bytecode are displayed in the screen. A deployer can connect to the wallet and sign the contract deployment transaction. 
**Backend**
Admin details are configured in the backend. The admin acts as a fee payer, signs the txn, pays for the txn and executes the txn to deploy the smart contract. 

```

File: feedelegation/README.md (hunk 7 above)
Comment (praveen-klaytn) [reply feedelegation/README.md]: removed web3modal title

**Review by toniya-klaytn — APPROVED** [review]

**Pull request merged**